<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-V1VAL2QI" failures="1" tests="1" name="Test3" time="156.035" errors="0" timestamp="29 Jan 2021 09:24:07 GMT">
  <testcase classname="TestCases.tatkalTicket" name="bookTicket" time="102.578">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at TestCases.tatkalTicket.bookTicket(tatkalTicket.java:308)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- bookTicket -->
</testsuite> <!-- Test3 -->
